ding_periodical.module

```diff
@@ -135,12 +135,14 @@ function theme_ding_periodical_issues($variables) {
 
       // Check if the volume is reservable and add reservation button.
       if (!empty($issue_id) && ding_periodical_is_reservable($issues[$key])) {
-        $path = array($issue_id, $volume, $key);
+        $item_id = array($issue_id, $volume, $key);
+        $item_id = array_map('base64_encode', $item_id);
```

Comment:
Adding a comment here explaining *why* we base64_encode the values would be good.